README.md

```diff
@@ -2,20 +2,29 @@
 
 Questions? [Get in touch][uportal-user@].
 
-<!-- current project status -->
+<!-- coordinates -->
 [![npm version](https://badge.fury.io/js/%40uportal%2Fapp-framework.svg)](https://badge.fury.io/js/%40uportal%2Fapp-framework)
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.apereo.uportal/uportal-app-framework/badge.svg)](https://maven-badges.herokuapp.com/maven-central/org.apereo.uportal/uportal-app-framework)
+<!-- build status -->
 [![Build Status](https://travis-ci.org/uPortal-Project/uportal-app-framework.svg)](https://travis-ci.org/uPortal-Project/uportal-app-framework)
-[![Coverage Status](https://coveralls.io/repos/uPortal-Project/uportal-app-framework/badge.svg?branch=master&service=github)](https://coveralls.io/github/uPortal-Project/uportal-app-framework?branch=master)
+<!-- dependencies statuss -->
```

Comment:
"status" is uncountable, i.e. it is both singular and plural.
https://www.macmillandictionary.com/dictionary/american/status def 3

Reply:
"stati" is definitely the way to go.

Reply:
Apparently ["statuses" is a thing per Merriam-Webster](https://www.merriam-webster.com/dictionary/status).